posix/unix.c

```diff
@@ -674,6 +674,11 @@ static ssize_t recv(unsigned socket, void *buf, size_t len, int flags, struct so
 		for (;;) {
 			proc_lockSet(&s->lock);
 			if (s->type == SOCK_STREAM) {
+				if (flags & MSG_PEEK) {
```

Comment:
MSG_PEEK applies to packet sockets as well. Please add TODO if you don't want to implement this case.

Reply:
Added a TODO